src/util.ts

```diff
@@ -87,15 +89,28 @@ export async function signAddress(string: string, privateKey: string) {
 
 /**
  * Generates a random string of the specified length
- * TODO: crypto.randomBytes
  */
-export function getRandomString(length: number) {
-	const chars = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789'
-	let result_str = ''
-	for (let i = 0; i < length; i++) {
-		result_str += chars[Math.floor(Math.random() * chars.length)]
+export function getRandomString(n: number = 18): string {
```

Comment:
The default here is not the same as the 16 in the code and creates some inconsistency.

I'd suggest having a function called generatePeanutKey which is called everywhere in index.ts that we call getRandom string and within this function have a single definition of how long the key should be. That way we have no chance of accidentally creating a random string that's too short.

Reply:
Inconsistency: yes. Thought making it slightly longer was perhaps preferrable after our discussion last week

generatePeanutKey: How would this look like? Is the suggestion to merge getRandomString and generateKeysFromString here?

e.g. this bit of code
```js

	if (passwords.length == 0) {
		passwords = Array(numberOfLinks).fill(getRandomString(16))
	}

	const keys = passwords.map((password) => generateKeysFromString(password)) // deterministically generate keys from password
```

And then just have one `generatePeanutKey`?

I think that's sensible, but lets not waste too much time on details!

Reply:
getRandomString(x) feels like an implementation detail and thus would be better not exposed.

generatePeanutKey() is the function that would be used everywhere in the SDK and exposed if needed -> this simply wraps a call to getRandomString with the length we expect. This means we don't have to have getRandomString(16) everywhere and if we change it have to remember to set 16 -> somethign else everywhere later. It's just better design I think